launch/camera_pipeline+machine.launch

```diff
@@ -1,12 +1,16 @@
 <launch>
 	<arg name="camera" default="camera"/>
 	<arg name="camera_id" default="0" />
+	<arg name="image_width" default="640" />
+	<arg name="image_height" default="480" />
 	<arg name="publish_rate" default="15"/>
 	<arg name="machine" default=""/>
 	<arg name="info_url" default=""/>
 
-	<node name="$(arg camera)" pkg="simple_webcam" type="camera.py" machine="$(arg machine)">
+	<node name="$(arg camera)" pkg="simple_webcam" type="camera.py" machine="$(arg machine)" output="screen" >
```

Comment:
That sounds like debugging to me. I'd prefer getting rid of it and letting people use `--screen` if they need to check.